### ISSUE TITLE
Additional notification type to change the reference power transmitted with iBeacon advertisements

### ISSUE THREAD

**Issue by rpatel3001** [home-assistant/android#3290]
**Is your feature request related to a problem? Please describe.**
The Wear OS app currently does not allow adjusting BLE beacon parameters except via notification. I would like to change the reference power to improve distance measurement accuracy.

**Describe the solution you'd like**
An additional command for message type command_ble_transmitter called ble_set_reference_power taking a float parameter ble_reference

**Describe alternatives you've considered, if any**
Ignoring the included power and using a custom value on the backend. Not ideal.

**Additional context**
The reference power is called the measured power in the phone app settings so s/reference/measured/ above if you want.

**Comment by agarbato** [comment]
I have the same exact need, change rssi1m as default -59 is completely wrong.
Even with station at 10cm espresence report 3 meters which won't work for all stations max distance settings.
Is it possible at least to have a default value as -80,-85 which is definitely more realistic for 1m distance (at least for all my other devices).
Thanks!

**Comment by dshokouhi** [comment]
> Is it possible at least to have a default value as -80,-85 which is definitely more realistic for 1m distance (at least for all my other devices).

a lot of research went into the current default value here: https://github.com/home-assistant/android/issues/1715

Either way thats not the same as this request so please file a separate request to discuss things, I will say be prepared to have some data to backup your findings compared to what was discovered in the issue

**Comment by agarbato** [comment]
Thanks @dshokouhi .
I found this already [#3359 ](https://github.com/home-assistant/android/issues/3359) which brought me here.
Smart watches have less power antennas (mine is a Ticwatch pro5). Still we have 3 android phones and we had to change setting to -80 -85 to make distance realistic.   Anyway I guess it depends on the device but for smart watches -59 1m does not really works so being able to change it via notification would be great :-)
